### PR TITLE
remove incubating or incubator word

### PR DESCRIPTION
**Pull request by jixuan1989**
this PR remove "incubator-" or "incubating" word.
And will test whether our travis-ci and coverall.io run well.

### CODE REVIEW

File: docs/zh/UserGuide/Get Started/Frequently asked questions.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ IoTDB> count timeseries root
 
 ## 可以使用 Hadoop 和 Spark 读取 IoTDB 中的 TsFile 吗？
 
-是的。IoTDB 与开源生态紧密结合。IoTDB 支持 [Hadoop](https://github.com/apache/incubator-iotdb/tree/master/hadoop), [Spark](https://github.com/apache/incubator-iotdb/tree/master/spark) 和 [Grafana](https://github.com/apache/incubator-iotdb/tree/master/grafana) 可视化工具.
+是的。IoTDB 与开源生态紧密结合。IoTDB 支持 [Hadoop](https://github.com/apache/iotdb/tree/master/hadoop), [Spark](https://github.com/apache/incubator-iotdb/tree/master/spark) 和 [Grafana](https://github.com/apache/incubator-iotdb/tree/master/grafana) 可视化工具.
```
Comment (HTHou) [inline docs/zh/UserGuide/Get Started/Frequently asked questions.md:133]:
```suggestion
是的。IoTDB 与开源生态紧密结合。IoTDB 支持 [Hadoop](https://github.com/apache/iotdb/tree/master/hadoop), [Spark](https://github.com/apache/iotdb/tree/master/spark) 和 [Grafana](https://github.com/apache/iotdb/tree/master/grafana) 可视化工具.
```

File: docs/zh/SystemDesign/TsFile/Format.md [hunk 2]
```diff
@@ -331,7 +331,7 @@ Linux or MacOs:
 
 在mac系统中的示例:
 
-```$xslt
+```console
 /incubator-iotdb/server/target/iotdb-server-0.10.0/tools/tsfileToolSet$ ./print-tsfile-sketch.sh test.tsfile
```
Comment (HTHou) [inline docs/zh/SystemDesign/TsFile/Format.md:335]:
```suggestion
/iotdb/server/target/iotdb-server-0.10.0/tools/tsfileToolSet$ ./print-tsfile-sketch.sh test.tsfile
```

File: README.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ IoTDB provides three installation methods, you can refer to the following sugges
 
 * Installation from source code. If you need to modify the code yourself, you can use this method.
 * Installation from binary files. Download the binary files from the official website. This is the recommended method, in which you will get a binary released package which is out-of-the-box.(Comming Soon...)
```
Comment (HTHou) [inline README.md:103]: Remove `Comming Soon...`?

File: docs/UserGuide/Server/Download.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ IoTDB provides you three installation methods, you can refer to the following su
 
 * Installation from source code. If you need to modify the code yourself, you can use this method.
 * Installation from binary files. Download the binary files from the official website. This is the recommended method, in which you will get a binary released package which is out-of-the-box.(Comming Soon...)
```
Comment (qiaojialin) [inline docs/UserGuide/Server/Download.md:27]:
```suggestion
* Installation from binary files. Download the binary files from the official website. This is the recommended method, in which you will get a binary released package which is out-of-the-box.

File: README.md (hunk 3 above)
Comment (qiaojialin) [inline README.md:103]:
```suggestion
* Installation from binary files. Download the binary files from the official website. This is the recommended method, in which you will get a binary released package which is out-of-the-box.

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**